scipy/spatial/qhull.pyx

```diff
@@ -555,6 +555,7 @@ cdef class _Qhull:
         cdef np.ndarray[np.npy_int, ndim=1] id_map
         cdef double dist
         cdef int facet_ndim
+        cdef np.npy_intp * coplanar_shape
```

Comment:
Unless it's by convention, you usually don't need to declare pointer variables because they are safely handled by the type inference.

Reply:
I've tried to remove the point variable type declarations now. Seems to work fine by inference as you say.